perfkitbenchmarker/scripts/database_scripts/launch_mysql_service.py

```diff
@@ -278,8 +278,8 @@ def _execute_pkb_cmd(pkb_cmd, stdout_filename, stderr_filename):
   # Will probably have to implement with threading.
   p.wait()
   elapsed_time = time.time() - start_time
-  if elapsed_time == 1:
-    raise CallFailureError('The call failed before execution (duration 1s). '
+  if p.returncode != 0:
```

Comment:
Yes. This is working for success cases. 

I've been trying to simulate a failure that doesn't cause a different exception to be thrown before reaching this one.